endoreg_db/models/media/video/video_file_segments.py

```
@@ -0,0 +1,196 @@
+
+            # Prepare data for LabelVideoSegment creation
+            init_dict = {
+                "video_file": video,
+                "prediction_meta": video_prediction_meta,
+                "label": label,
+                "start_frame_number": start_frame_number,
+                "end_frame_number": end_frame_number,
+            }
+            try:
+                # Assuming custom_create handles creation and saving or returns unsaved instance
+                segment = LabelVideoSegment.custom_create(**init_dict)
```

Ruff (0.8.2)

Local variable `segment` is assigned to but never used

Remove assignment to unused variable `segment`

(F841)